community_build.sh

```diff
@@ -0,0 +1,146 @@
+# COMMUNITY_BUILD.SH notes: 
```

Comment:
Let's make sure we include a shebang here to make sure the behavior is the same for various users.